src/inputs/Switch.tsx

```diff
@@ -42,27 +46,30 @@ export function Switch(props: SwitchProps) {
     <label
       {...hoverProps}
       css={{
-        ...switchLabelDefaultStyles,
```

Comment:
I inlined a bunch of these styles that were constants b/c the idea of tachyons/utility css is to have the css defined inline / as close to the markup as possible.